android/app/src/main/AndroidManifest.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" >
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
     <application
-        android:usesCleartextTraffic="true"
```

Comment:
need to evaluate if i need this anymore

Reply:
im going to leave this out, but I think we may need it for Jakub, so can re-introduce later